### PR TITLE
Mapped out a few resource-related stuff

### PR DESCRIPTION
**Pull request by liach**
I need them for my mod! Please take it!

Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

File: mappings/net/minecraft/client/MinecraftClient.mapping [hunk 1]
```diff
@@ -141,5 +143,6 @@ CLASS cof net/minecraft/client/MinecraftClient
 	METHOD s isFancyGraphicsEnabled ()Z
 	METHOD t isAmbientOcclusionEnabled ()Z
 	METHOD u getInstance ()Lcof;
+	METHOD v reloadConcurrently ()Ljava/util/concurrent/CompletableFuture;
```
Comment (asiekierka) [inline mappings/net/minecraft/client/MinecraftClient.mapping]: reloadResourcesConcurrently?

File: mappings/net/minecraft/client/resource/ClientResourcePackCreator.mapping [hunk 2]
```diff
@@ -0,0 +1,37 @@
+CLASS dme net/minecraft/client/resource/ClientResourcePackCreator
```
Comment (asiekierka) [inline mappings/net/minecraft/client/resource/ClientResourcePackCreator.mapping:1]: Creator? Not Downloader?

**Review by asiekierka — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/resource/ClientResourcePackCreator.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/client/resource/ClientResourcePackCreator.mapping:1]: This also creates resource packs for the default pack and the old programmer art pack.

**Review by asiekierka — APPROVED** [review]

File: mappings/net/minecraft/client/MinecraftClient.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/client/MinecraftClient.mapping]: Fix in next push.

**Review by modmuss50 — APPROVED** [review]

**Comment by liach** [comment]
thanks!

**Pull request merged**